app/views/ppm_estimator/index.html.erb

```diff
@@ -0,0 +1,154 @@
+<%- @page_title = 'Personally Procured Move (PPM) Estimator' -%>
+
+<div class="usa-section main-section">
+  <h1><%= @page_title %></h1>
+
+  <h4>
+    For your upcoming move, get an estimate for how much the government will
+    pay <strong>you</strong> if you choose to "do it yourself."
```

Comment:
i think the period should be outside the quotes?

Reply:
> the answer depends on where your audience lives because in American English we always put periods and commas inside quotation marks

- http://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks